Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:14.18.0-stretch-slim
```

Comment:
The _node:14.18.0-alpine_ image is smaller by 50 MB, but I could not successfully install the [gif2webp-bin](https://github.com/imagemin/gif2webp-bin) package.

I have tried different versions of Alpine, as well as different versions of Node.js, but always get the same error:

![gif2webp-bin](https://user-images.githubusercontent.com/9103819/151563365-111282f9-a9a9-4f81-bbff-5f504d28da37.png)

Found an issue https://github.com/sindresorhus/got/issues/1572, but it doesn't seem to contain a solution to this problem 🤷‍♂️

Reply:
Slim is fine, I think.